src/blocks/kpi-chart/index.tsx

```
@@ -0,0 +1,303 @@
+function alignStartDate(date: Date, periodType: PeriodType): Date {
+	const aligned = new Date(date.getTime());
+
+	switch (periodType) {
+		case "week":
+			// Weeks start on Monday (day 1)
+			// getDay() returns 0 (Sun),1(Mon),2(Tue),3(Wed),4(Thu),5(Fri),6(Sat)
+			// To get Monday, we find how many days we need to go back:
+			// If day = 0 (Sun), go back 6 days
+			// If day = 1 (Mon), go back 0 days
+			// If day = 2 (Tue), go back 1 day, etc.
+			const dayOfWeek = aligned.getDay();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)